### PR TITLE
Set LDAP protocol as env parameter. Default value: ldap.

### PR DESCRIPTION
**Pull request by marisbahtins**
At the moment LDAP protocol in nginx.conf is hardcoded. To avoid that I made env variable which by default is "ldap" but if there is a need to use other protocol we also can do this using this environment variable.

### CODE REVIEW

**Comment by svscorp** [comment]
How is it different from #42 where LDAP(s) is implemented?

**Comment by SachinKSingh28** [comment]
@svscorp I believe it would be easy to test this PR which includes only LDAP protocol related changes. My proposal is that we merge this change and rebase  changes on https://github.com/Accenture/adop-nginx/pull/42 while review for that PR is in progress. What do you think?

File: README.md [hunk 1]
```diff
@@ -19,23 +19,24 @@ adop-nginx provides Nginx with LDAP support. Nginx is a high performance reverse
...
-        
+
 ## Configuration
 
 The nginx configuration is externalised and stored the 'resources' directory.
 
 Runtime configuration can be provided using environment variables:
 
 * LDAP_SERVER, the LDPA URI, i.e. ldap-host:389
+* LDAP_PROTOCOL, ldap(default) or ldaps
```
Comment (SachinKSingh28) [inline README.md]: Can we please add some description for this parameter along with the default value?

**Comment by SachinKSingh28** [comment]
Looks good.

**Pull request merged**